docs/book/src/reference/markers/crd.md

```diff
@@ -0,0 +1,7 @@
+# CRD Construction
+
+These markers describe how to construct a custom resource definition from
+a series of Go types and packages.  Generation of the actual validation
+schema is described by the [validation markers](./crd-validation.md).
+
+{{#markerdocs CRD}}
```

Comment:
that's intentional -- otherwise you get some things indented and others not, and it's hard to read the list vertically -- swing by and I'll show you what I mean